js/src/Dapps/dapps.js

```diff
@@ -50,19 +50,17 @@ class Dapps extends Component {
     const applications = [].concat(this.store.visibleLocal, this.store.visibleViews, this.store.visibleBuiltin, this.store.visibleNetwork);
 
     return (
-      <Page>
-        <div className={ styles.dapps }>
-          {
-            applications.map((app, index) => (
-              <DappCard
-                app={ app }
-                availability={ availability }
-                className={ styles.dapp }
-                key={ `${index}_${app.id}` }
-              />
-            ))
-          }
-        </div>
+      <Page className={ styles.dapps }>
```

Comment:
I'm modifying this heavily in #7266, I think you can remove it from this PR. It would save some time for me or you resolving conflicts

Reply:
Yup, just made a comment - think we need this one in first and then make sure that your still works 100% as expected. I'm worried since the top layout is now not a fixed bar anymore, rather a proper layout. (Needed to get done since we have had complaints about overlaps)

Reply:
Agreed, i'll rebase mine on yours.